packages/experimental/Tabs/src/useTabsItem.ts

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import { useAsPressable, useKeyCallback, useOnPressWithFocus, useViewCommandFocus } from '@fluentui-react-native/interactive-hooks';
+import { TabsItemProps, TabsItemInfo } from './TabsItem.types';
+import { TabsContext } from './Tabs';
+
+export const useTabsItem = (props: TabsItemProps): TabsItemInfo => {
+  // Attach the pressable state handlers.
+  const defaultComponentRef = React.useRef(null);
+  const { accessibilityLabel, headerText, componentRef = defaultComponentRef, itemKey, disabled, itemCount, icon, ...rest } = props;
+  // Grabs the context information from Tabs (currently selected TabsItem and client's onTabsClick callback).
+  const info = React.useContext(TabsContext);
+  const tabsItemSelectActionLabel = 'Select a TabsItem';
```

Comment:
What resulted from the localization conversation that happened for the other PR? Hardcoding this into English seems like it wouldn't be very user friendly for other locales.

Reply:
I haven't looked further into localization since the last conversation but I'll create an issue to track it. I'm not sure this string is even used by assistive technology (at least on win32) so it may never reach the end user but I'll double check.

Reply:
I would not put any English strings directly in the component. If we want to test it, I would prefer it's in a test page inside FluentTester.

Reply:
The accessibility action label is optional and it's not hooked up for win32 anyways so yeah let's remove it.